app/institutions/dashboard/-components/institutional-users-list/template.hbs

```diff
@@ -204,4 +212,51 @@
             {{t 'institutions.dashboard.users_list.empty'}}
         </list.empty>
     </PaginatedList::HasMany>
+    <OsfDialog @isOpen={{this.messageModalShown}} @onClose={{this.toggleMessageModal}} as |dialog|>
+        <dialog.heading>
+            {{t 'institutions.dashboard.send_message_modal.title'}}
+        </dialog.heading>
+        <dialog.main>
+            <div>
+                <label for='message-text' local-class='message-label'>
+                    {{t 'institutions.dashboard.send_message_modal.opening_message_label'}}
+                </label>
+                <Textarea
+                  id='message-text'
+                  local-class='message-textarea'
+                  @value={{this.messageText}}
+                />
+                <div>
+                    {{t 'institutions.dashboard.send_message_modal.closing_message_label' adminName=this.currentUser.user.fullName htmlSafe=true}}
+                </div>
+                <div local-class='checkbox-container'>
+                    <label local-class='checkbox-item'>
+                        <input type='checkbox' checked={{this.cc}} {{on 'change' this.toggleCc}} />
+                        {{t 'institutions.dashboard.send_message_modal.cc_label'}}
+                    </label>
+                    <label local-class='checkbox-item'>
+                        <input type='checkbox' checked={{this.replyTo}} {{on 'change' this.toggleReplyTo}} />
+                        {{t 'institutions.dashboard.send_message_modal.reply_to_label'}}
+                    </label>
```

Comment:
Similar to the `<Textarea>`above, you can use ember's built-in `<Input @type='checkbox' @checked={{this.cc}}>` in the above two cases. This way you get the value-binding automatically, and won't need the `toggleCc` and `toggleReplyTo` actions